test/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
What does this do? Config eslint for these dirs, right? I'm looking at the GUI one and don't understand the line:
```
    extends: ['scratch/react', 'scratch/es6'],

```
...is this doing something we should do here?

Reply:
@rschamp worked with me on getting this part to work. I believe the linter needed the `env: {jest: true}` part to allow the test files to pass the linter.  Without it it has a lot of "test was used but never defined" and similar errors since we're using many functions defined in Jest.

 I didn't add the other parts from gui because they weren't necessary. They must pertain to other files in the gui tests.